sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/models/RoomParticipant.java

```diff
@@ -23,7 +23,7 @@ public final class RoomParticipant {
      * The Role of a room participant.
      */
     @JsonProperty(value = "role")
-    private RoleType role;
+    private Role role;
```

Comment:
Create new `InvitedRoomParticipant` model which is taken as the parameter and `RoomParticipant` is used as the return value